keras_nlp/src/layers/preprocessing/masked_lm_mask_generator_test.py

```diff
@@ -12,14 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-try:
-    import tensorflow as tf
-except ImportError:
-    raise ImportError(
-        "To use `keras_nlp`, please install Tensorflow: `pip install tensorflow`. "
-        "The TensorFlow package is required for data preprocessing with any backend."
-    )
-
+import tensorflow as tf
```

Comment:
Don't we need a [try/catch for tf export here (like other files)](https://github.com/keras-team/keras-nlp/pull/1660/files#diff-ff946da9b249ce45c338c3f5402c3dddfe3622edea7027f1424470ad240de4d0R23-R26)?

Reply:
I was thinking to let an unconditional `import tensorflow` be fine for test code. This is true for core Keras too. There's really no benefit to allowing our test suite to run without tensorflow.

For library the benefit is clear, if niche. This allows running our layers, models with preprocessing from another library.

Basically, this all bring us close to core Keras behavior, which is probably the right thing.

Reply:
And the nice thing about adding a test is we don't have to remember any of this. If you add an unconditional import in the wrong place -> tests go red. So we won't have to do any constant bookkeeping during review.